src/components/ProductList.tsx

```diff
@@ -70,7 +69,14 @@ const ProductList = () => {
       <div className="mb-8">
         <ProductSearch onSearch={handleSearch} />
       </div>
-      
+
+      {/* مكون جديد لعرض عدد المنتجات */}
+      <div className="mb-6 text-right text-slate-700 mt-6">
+        <p>
+          {t("عدد المنتجات التي تظهر هو")}: <span className="font-bold">{filteredProducts.length}</span>
```

Comment:
Nice addition! However, to keep the UI simple for now, let's hold off on this. We can revisit this idea later and add it in a small place if needed.